.github/workflows/build-and-deploy.yml

```
@@ -41,7 +41,7 @@
   upload-release-asset:
     needs: build # This job depends on the build job
     runs-on: ubuntu-latest # Use the latest Ubuntu runner
-    if: github.event_name == 'release' # Only run this job on release events
+    #if: github.event_name == 'release' # Only run this job on release events
```

yamllint

[warning] missing starting space in comment

(comments)